app/Resources/views/articleInfo/result.html.twig

```diff
@@ -42,7 +57,7 @@
             {% if ai.assessments and not (page.isMainPage) %}
                 {% set sections = sections|merge(['assessments']) %}
             {% endif %}
-            {% if ai.numBugs > 0 %}
+            {% if ai.numBugs > 0 and not (ai.hasDateRange) %}
```

Comment:
Let's do `not (ai.hasDateRange)` first, because `ai.numBugs` will run the queries.

Reply:
Also need the `not (ai.hasDateRange)` check on line 57